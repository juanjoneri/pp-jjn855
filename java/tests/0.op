-action 0
-sum 0
-when ($1==8)||($1==y)
-print 2,2,2,2,0
