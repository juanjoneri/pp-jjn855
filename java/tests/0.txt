A B C
1 3 0
b 4 asd
aa 10 ab
10 10 aasd