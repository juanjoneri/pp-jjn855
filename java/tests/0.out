1 5
2 7
22 50
